sklearn/discriminant_analysis.py

```diff
@@ -978,11 +1057,6 @@ def _decision_function(self, X):
     def decision_function(self, X):
         """Apply decision function to an array of samples.
 
-        The decision function is equal (up to a constant factor) to the
-        log-posterior of the model, i.e. `log p(y = k | x)`. In a binary
-        classification setting this instead corresponds to the difference
-        `log p(y = 1 | x) - log p(y = 0 | x)`. See :ref:`lda_qda_math`.
```

Comment:
This kind of information is now removed from the docstrings. I think we should put back what the decision function does.